plugins/amqp_trace_plugin/amqp_trace_plugin.cpp

```diff
@@ -135,7 +135,7 @@ void amqp_trace_plugin::plugin_shutdown() {
 
          my->applied_transaction_connection.reset();
          if( my->amqp_trace ) {
-            my->amqp_trace->stop();
+            my->amqp_trace.reset();
```

Comment:
Does this create a problem if the `trace_plugin` shuts down before the `trx_plugin`?

Reply:
Yes. I forgot that is why I had it that way. Reverted.

Reply:
It might be a good idea to add a comment to make sure that no one else forgets it.

Reply:
Added comment.